counter/src/main/java/org/infinispan/counter/impl/manager/CounterConfigurationStorage.java

```diff
@@ -28,6 +28,13 @@
     */
    void store(String name, CounterConfiguration configuration);
 
+   /**
+    * Remove a counter configuration
+    *
+    * @param name the counter's name.
+    */
+   void remove(String name);
```

Comment:
We called `removeConfiguration` in the CounterConfigurationManager.java. maybe keep the name here as well?

Reply:
I think since this class is named ```CounterConfigurationStorage```, and it already has a ```store``` and ```load``` methods, calling it ```removeConfiguration``` here would make it a bit off